### PR TITLE
osd: Remove unnecessary copy, parameter declaration

### PR DESCRIPTION
**Pull request by shinobu-x**
Signed-off-by: Shinobu Kinjo <shinobu@redhat.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
retest this please.

**Comment by shinobu-x** [comment]
test this please

**Comment by shinobu-x** [comment]
test this please

**Comment by shinobu-x** [comment]
test this please

**Comment by tchaikov** [comment]
retest this please.

**Comment by tchaikov** [comment]
retest this please.

File: src/osd/OSD.cc [hunk 1]
```diff
@@ -7148,10 +7148,8 @@ vector<OSDHealthMetric> OSD::get_health_metrics()
 {
   vector<OSDHealthMetric> metrics;
   {
-    utime_t oldest_secs;
-    const utime_t now = ceph_clock_now();
-    auto too_old = now;
-    too_old -= cct->_conf->get_val<double>("osd_op_complaint_time");
+    auto too_old =
+      ceph_clock_now() - cct->_conf->get_val<double>("osd_op_complaint_time");
```
Comment (tchaikov) [inline src/osd/OSD.cc]: good.

File: src/osd/OSD.cc [hunk 2]
```diff
@@ -7161,7 +7159,10 @@ vector<OSDHealthMetric> OSD::get_health_metrics()
 	return false;
       }
     };
-    if (op_tracker.visit_ops_in_flight(&oldest_secs, count_slow_ops)) {
+    utime_t oldest_secs;
+    if (slow != 0) {
+      utime_t oldest_secs;
```
Comment (tchaikov) [inline src/osd/OSD.cc]: this variable shadows the outer one.

File: src/osd/OSD.cc [hunk 3]
```diff
@@ -7161,7 +7159,10 @@ vector<OSDHealthMetric> OSD::get_health_metrics()
 	return false;
       }
     };
-    if (op_tracker.visit_ops_in_flight(&oldest_secs, count_slow_ops)) {
+    utime_t oldest_secs;
+    if (slow != 0) {
```
Comment (tchaikov) [inline src/osd/OSD.cc]: slow is always 0 here.

**Comment by jdurgin** [comment]
Closing due to unaddressed comments.

**Pull request closed without merging**